### PR TITLE
Adds MBS schemas

### PR DESCRIPTION
**Pull request by nealedj**
Adds all MBS schemas

(squashed because there was an extra linting commit I had to make to the feature branch)

### CODE REVIEW

**Comment by nealedj** [comment]
Could somebody please review that routing rule fix?

File: data/en/mbs_0251.json [hunk 1]
```diff
@@ -581,7 +581,7 @@
                             },
                             {
                                 "goto": {
-                                    "group": "summary-section"
```
Comment (shahi645) [inline data/en/mbs_0251.json]: @nealedj Just tested and this change works fine. Just to confirm though, in this schema employees section comes after significant changes? as in all other employee schemas for mbs this is the opposite 😕

**Comment by shahi645** [comment]
All passed schema-validator locally 👍

**Comment by davecharles** [comment]
LGTM

**Review by davecharles — APPROVED** [review]
LGTM

**Review by shahi645 — APPROVED** [review]

**Pull request merged**